src/eclingo/internal_states/internal_control.py

```diff
@@ -67,6 +70,12 @@ def __init__(
             clingox_program.ProgramObserver(self.ground_program)
         )
 
+        # HERE add the new observer for the Reifier
+        self.reification = config.eclingo_reification
+        reify_terms: List[Symbol]
+        if self.reification:
+            self.control.register_observer(Reifier(reify_terms.append))
+
```

Comment:
I apologize, I forgot to push the changes I was mentioning. 

@jorgefandinno Could you check whenever you have time the old comment and point in the direction to follow. I thought I had the idea developed, but I'm struggling to see how I should connect what I did on the last test into the internal states controller.

Thanks,

PD: A reminder that I'm in Denver and I'll be missing Wednesday 9th lab meeting.

Reply:
Btw, did we get clarification in what tag(incremental) was?